canonical-websites/partners.ubuntu.com
# remove mobile

## Done

* removed /programmes/phone
* added redirect to /
* updated homepage copy and image
* removed intro from /programmes page
* removed from the nav

## QA

1. on /programmes/phone see that it redirects to /
2. on /programmes see that there is no phone box, no phone in nav
3. on / see that the phone section now has a robot and new text per the [copy doc](https://docs.google.com/document/d/151x2H8xmKQzy-gwEUlijTRbzyfnPomaVV00zjwbNW4g/edit#heading=h.2ia35mgzzw8f)
